editoast/src/modelsv2/projects.rs

```diff
@@ -118,84 +118,101 @@ impl Project {
 #[cfg(test)]
 pub mod test {
     use rstest::rstest;
-    use std::sync::Arc;
+    use std::ops::DerefMut;
```

Comment:
```suggestion
    use std::ops::DerefMut;
    use pretty_assertions::assert_eq;
```

It's a drop-in replacement for `assert_eq!`, but shows a colored diff on failure. It's already in the dependance tree so we might as well use it.